### DISCUSSION TITLE
Scratch Port

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#2031] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Can someone pls make a port for Scratch 
 

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
what do you mean by scratch?

**Reply by sgoudham** [reply]
I assume they mean https://scratch.mit.edu/projects/editor/

**Reply by ghost** [reply]
Yes

**Comment by Ankh64** [comment]
i think you can do this by getting this extension https://github.com/ScratchAddons/ScratchAddons and modifying the colors in the editor dark mode addon.
